airflow/providers/google/cloud/hooks/dataprep.py

```diff
@@ -25,25 +25,25 @@
 from airflow import AirflowException
 from airflow.hooks.base_hook import BaseHook
 
-URL = "https://api.clouddataprep.com/v4/jobGroups/"
-
 
 class GoogleDataprepHook(BaseHook):
     """
     Hook for connection with Dataprep API.
+    To get connection Dataprep with Airflow you need Dataprep token.
+    https://clouddataprep.com/documentation/api#section/Authentication
+
+    It should be added to the Connection in Airflow in JSON format.
```

Comment:
Should we add a reference to howto guide with example?

Reply:
Done. Will you check?